irohad/ametsuchi/impl/rocksdb_command_executor.hpp

```diff
@@ -62,159 +64,179 @@ namespace iroha::ametsuchi {
         shared_model::interface::types::CommandIndexType cmd_index,
         bool do_validation) override;
 
-    CommandResult operator()(
+    ExecuteResult operator()(
```

Comment:
Too much common parameters between similar functions. Let's refator. Do not use operator().